src/extensions/arithmetic/conversions.rs

```diff
@@ -24,7 +24,7 @@ fn ftoi_sig(arg_values: &[TypeArg]) -> Result<(TypeRow, TypeRow, ResourceSet), S
     let [arg] = collect_array(arg_values);
     let n: u8 = get_width(arg)?;
     Ok((
-        vec![float64_type()].into(),
+        vec![FLOAT64_TYPE].into(),
```

Comment:
now I'm just getting annoying but...
```suggestion
        type_row![FLOAT64_TYPE],
```

Reply:
Thanks, lots of instances of that. Replaced.